pkg/agent/openflow/client.go

```diff
@@ -174,13 +173,9 @@ func (c *client) InstallNodeFlows(hostname string,
 ) error {
 	c.replayMutex.RLock()
 	defer c.replayMutex.RUnlock()
-	flows := make([]binding.Flow, 2, 3)
-	flows[0] = c.arpResponderFlow(peerGatewayIP, cookie.Node)
-	flows[1] = c.l3FwdFlowToRemote(localGatewayMAC, peerPodCIDR, tunnelPeerAddr, tunOFPort, cookie.Node)
-	if tunnelPeerAddr == nil {
-		// Not the default (flow based) tunnel. Add a separate tunnelClassifierFlow.
-		flows = append(flows, c.tunnelClassifierFlow(tunOFPort, cookie.Node))
```

Comment:
I have one question, since a seperate tunnel port for each node is created, which is used for OVS IPSec monitor, and I found that no Openflow entries are installed for these special ports, is it possible some packets need to be transmitted by these ports? If so, as there is no Openflow entries for these special ports, the packet should be dropped at last. If no packets processed by these port, please ignore my question.

Reply:
Yes, no packet will really go to / come from these tunnel ports. They are just for OVS IPSec monitor to detect a tunnel and create security policies - you can say this is not the best way though.